src/System.Management.Automation/engine/NativeCommandProcessor.cs

```diff
@@ -1108,14 +1125,14 @@ private void ProcessOutputRecord(ProcessOutputObject outputValue)
         }
 
         /// <summary>
-        /// Gets the start info for process.
+        /// Gets the ProcessStartInfo for process.
         /// </summary>
-        /// <param name="redirectOutput"></param>
-        /// <param name="redirectError"></param>
-        /// <param name="redirectInput"></param>
-        /// <param name="soloCommand"></param>
-        /// <returns></returns>
-        private ProcessStartInfo GetProcessStartInfo(bool redirectOutput, bool redirectError, bool redirectInput, bool soloCommand)
+        /// <param name="redirectOutput">A boolean which determines whether the output is to be redirected.</param>
```

Comment:
```suggestion
        /// <param name="redirectOutput">If true, redirect output from the process to a stream, otherwise send it to stdout.</param>
```